tools/ovc/openvino/tools/ovc/convert_impl.py

```diff
@@ -86,10 +94,19 @@ def arguments_post_parsing(argv: argparse.Namespace):
         print_argv(argv)
 
     params_parsing(argv)
-    argv.output = argv.output.split(',') if isinstance(argv.output, (str, pathlib.Path)) else argv.output
-
     log.debug("Placeholder shapes : {}".format(argv.placeholder_shapes))
 
+    if not hasattr(argv, 'output') or argv.output is None:
+        return argv
+
+    if argv.is_python_api_used:
+        error_msg = f"output '{argv.output}' is incorrect, it should be string or a list of strings"
+        assert isinstance(argv.output, (str, list)), error_msg
```

Comment:
List or tuple. We use them interchangeably in these parameters. @popovaan applied "list or tuple" feature for `input` parameter, please find it and make similar changes here.

Reply:
relaxed type assert, tuples are also allowed